stytch/oauth.go

```diff
@@ -55,3 +55,16 @@ type ProviderValues struct {
 	ExpiresAt    *time.Time `json:"expires_at,omitempty"`
 	Scopes       []string   `json:"scopes,omitempty"`
 }
+
+type OAuthAttachParams struct {
+	Provider     string `json:"provider,omitempty"`
+	UserID       string `json:"user_id,omitempty"`
+	SessionToken string `json:"session_token,omitempty"`
+	SessionJWT   string `json:"session_jwt,omitempty"`
```

Comment:
Let's indicate that exactly one of these fields should be filled in

Reply:
d1797bb2e676224ea693c4c605e52b25546643af